### PR TITLE
Blog Post ready

### PR DESCRIPTION
**Pull request by mduesterhoeft**
@BirgitBader  can you do the review before your vacation or should I rather ask Kristof and Steffi?

### CODE REVIEW

File: _posts/2015-08-11-one-month-as-an-epagee.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Our first days were packed with different information sessions and trainings. Many departments introduced themselves and gave some insight about their work. This really helped to understand the company. One of the highlights was certainly the Scrum Training given by Anja. The session was a nice refresher regarding the principles of scrum and how they are implemented at ePages. Anja also gave us some interesting information about the transformation of ePages from a waterfall-oriented company to an agile one.
+
+{% image blog-one-month-as-an-epagee-scrum.jpg %}{% endimage %}
+
+Another session we really enjoyed was the introduction of the ePages product held by André. He gave us an overview about the features of the product and also highlighted the strength and weaknesses. This was very helpful but the most interesting part of his session was when André talked about his experience from the ePages academy. The [ePages academy](|http://www.ePages.com/academy/en/) is series of events for our merchants that is about coaching and networking. So André could tell us what our merchants are doing with our product and could show some example shops. This was very valuable information .
```
Comment (BirgitBader) [inline _posts/2015-08-11-one-month-as-an-epagee.md]: Remove whitespace between information and fullstop.

File: _posts/2015-08-11-one-month-as-an-epagee.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Besides the training and introduction sessions there was not much room for being productive. So it was really time to join our teams full-time.
+
+Not surprisingly the first couple of weeks working with the team were packed with new information. The first meetings we attended where filled with topics, technologies and internal terms we haven't heard of. But the colleagues where very helpful and friendly, answering all our questions patiently. After a month at ePages there are still questions raising, but it feels like a deeper understanding of all the topics has started to grow and keeps increasing every day.
```
Comment (BirgitBader) [inline _posts/2015-08-11-one-month-as-an-epagee.md]: change to: But the colleagues were ...

**Comment by BirgitBader** [comment]
@maletzke please give your feedback @RealMattMurdoch please proof-read

File: _posts/2015-08-11-one-month-as-an-epagee.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+date: "2015-08-11 08:00:00"
+icon: user-plus
+tags: induction employee welcome introduction onboarding
+categories: jobs
+authors: ["Mathias", "Mats"]
+---
+
+Starting a new job is always an adventure. You made a careful decision for a new job and now the first day is approaching. This day is especially exciting because it will give you the first impression if the decision for this particular position was the right one.
+
+So it was really a little relief to receive a very warm welcome on our first day. Dirk and Harm said hello from the management side and introduced ePages. Leana from HR made sure that all the organisational stuff was taken care of and Keun-Man provided us with the hardware. So we were basically ready to start. Afterwards our buddies introduced us to everybody - ePages is a very diverse company. The team is very international and has a good mix of age groups. You find people from Spain, Italy, Belarus, Brasil, people that are on their first job and also more experienced ones - and even parents ;-).
```
Comment (maletzke) [inline _posts/2015-08-11-one-month-as-an-epagee.md]: "on our first day" <-- you didn't mention that there were others who started with you. Suggestion: "So when me and three other new employees had our first day, it was really a little relief to receive a very warm welcome."

Comment (maletzke) [inline _posts/2015-08-11-one-month-as-an-epagee.md]: "Dirk and Harm" --> "Dirk Jovy (VP Operations) and Harm Behrens (CTO)"

File: _posts/2015-08-11-one-month-as-an-epagee.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
```
Comment (maletzke) [inline _posts/2015-08-11-one-month-as-an-epagee.md]: "Anja" --> "usability expert Anja Gehl" It was Anja Gehl, right?

File: _posts/2015-08-11-one-month-as-an-epagee.md (hunk 1 above)
Comment (maletzke) [inline _posts/2015-08-11-one-month-as-an-epagee.md]: "André" --> "ecommerce trainer André Podeyn"

Comment (maletzke) [inline _posts/2015-08-11-one-month-as-an-epagee.md]: "and also highlighted the strength and weaknesses" --> i would delete this part

Comment (maletzke) [inline _posts/2015-08-11-one-month-as-an-epagee.md]: "is series of events" --> "is a series of events"

**Pull request merged**